Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/NativeArrayTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using NUnit.Framework;
+using Unity.Collections;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class NativeArrayTests : TypeTesterBase
+    {
+        public static (NativeArray<int> deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)